include/albatross/src/models/sparse_gp.hpp

```diff
@@ -117,6 +117,8 @@ struct StateSpaceInducingPointStrategy {
  * get those deails straight:
  *
  *     - https://bwengals.github.io/pymc3-fitcvfe-implementation-notes.html
+ *        (beware, the final formulae are correct but there are typos in
+ *         their derivations).
```

Comment:
Should this comment be dated? The referenced post could be corrected in the future.

Reply:
done